code/clue.js

```diff
@@ -43,22 +166,57 @@ const randomSelector = array => {
 // With a killer, a weapon and a room.
 // The values will be set later.
 
+const mystery = {
+};
```

Comment:
Are you using this object somewhere?

Reply:
This is the object that all get stored in, before that data or input gets printed on the screen in the h1 element when pushing the button, have I put it in the wrong order maybe?

Reply:
Okay! No it's fine just wanted to know how you use it! Cool 😁